### ISSUE TITLE
The questo for a (new) name

### ISSUE THREAD

**Issue by audetto** [audetto/AppleWin#165]
I have never like the name of the main executable `sa2`, standing for `SDL Apple 2`.

Since I am building a snapcraft package, I have decided to rethink it.

My current preference is for

`ca2e`: Classic Apple 2 Emulator

Why classic? Because it offers a good acronym.
Alternatives include `eXtended`, `Virtual` or modifications of the above like `a2ce`, but so far my preference is above.

**Comment by sh95014** [comment]
I feel like "yet another" should belong to newer emulators, not a port of a very old one and almost "default" one like AppleWin.

How about something like "AW4L", meaning AppleWin for Linux? Or even "AppleLin"?  😊

**Comment by xandark** [comment]
Yes, sa2 is a bit of a stretch to associate as an Apple II emulator. `ya2e` doesn't quite feel right.

How about `a2em4lin`? `aw4lin`?
